source/core/parameters_multiphysics.cc

```diff
@@ -541,6 +551,76 @@ Parameters::VOF_AlgebraicInterfaceReinitialization::parse_parameters(
   prm.leave_subsection();
 }
 
+void
+Parameters::VOF_GeometricInterfaceReinitialization::declare_parameters(
+  dealii::ParameterHandler &prm)
+{
+  prm.enter_subsection("geometric interface reinitialization");
+  {
+    prm.declare_entry(
+      "enable",
+      "false",
+      Patterns::Bool(),
+      "Enables the interface reinitialization with the geometric method "
+      "<true|false>");
```

Comment:
Would it not be better to  instead a parameter which would be:
retinitialization method = "none, pde, projection, geometric" and then enter that section?
It's not the best idea to have a ton of enable / everywhere and to rely on checking. This way we would have less parameters.

Reply:
I agree. Do I do it in this PR or I open a refactoring one? @blaisb

Reply:
I'll merge the PR as is now, this could be in another PR?